plugins/gin/config.go

```diff
@@ -0,0 +1,23 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package gin
+
+//skywalking:config gin
+var config struct {
+	CollectHttpHeaders string `config:"collect_http_headers"`
```

Comment:
A small suggestion, could you change his data type to `[]string`? Although it may be a bit challenging, you might need to provide parsing capabilities in the agent core, so that you don't have to use `runtimeConfig` and reprocess it for each request. If it's still difficult for you, I can help with it. :)

Reply:
Yes, that is bothering me. And I want to solve that. where should I getting start?

Reply:
You need to follow these steps:
1. Adding the `[]string` case in there: https://github.com/apache/skywalking-go/blob/main/tools/go-agent/instrument/plugins/enhance_config.go#L219-L240
2. Adding `ParseStringArray` into the [ToolOperator interface](https://github.com/apache/skywalking-go/blob/main/plugins/core/operator/tools.go) and [implement it](https://github.com/apache/skywalking-go/blob/main/plugins/core/tracer_tools.go).

Reply:
Working on it.